infrastructure/demo.tfvars

```diff
@@ -9,3 +9,5 @@ orchestrator_notifications_task_delay = "3000"
 api_gateway_test_valid_certificate_thumbprint = "C4784AD48B4B99F427D6B56FB38184D0E6457744"
 blob_signature_verification_key_file = "nonprod_public_key.der"
 allowed_client_certificate_thumbprints = ["4AF638E82FBD9959A5B8286C673360AC2C2E7053"]
+
+smtp_host = "smtp.office365.com"
```

Comment:
Does this determine if the reports are enabled or not. Is there any other way we can do this else let's add a comment here

Reply:
Same as feign clients are conditioned